deb/debian/stackdriver-agent.init

```diff
@@ -179,7 +137,12 @@ d_start() {
         export https_proxy=$PROXY_URL
     fi
 
-    if ! gen_config || ! check_config; then
+    if ! check_credentials || ! check_config; then
+        log_failure_msg "not starting, credentials error"
+        return 3
+    fi
+
+    if ! check_config; then
```

Comment:
I'm not sure how this condition wouldn't be captured by the condition above.

Reply:
Nope that was a bug. Fixed.